cmd/describe/cluster/cmd.go

```diff
@@ -335,6 +335,17 @@ func run(cmd *cobra.Command, argv []string) {
 			"OIDC Endpoint URL:          %s (%s)\n", str,
 			cluster.AWS().STS().OIDCEndpointURL(), managementType)
 	}
+	if cluster.AWS().PrivateHostedZoneID() != "" {
+		str = fmt.Sprintf("%s"+"Private Hosted Zone:\n", str)
+		str = fmt.Sprintf("%s"+
+			" - ID:                      %s\n", str,
+			cluster.AWS().PrivateHostedZoneID())
+		if cluster.AWS().PrivateHostedZoneRoleARN() != "" {
```

Comment:
This check is redundant, upon cluster creation we validate that both `privateHostedZoneID` and `privateHostedZoneRoleArn` are provided, so if `PrivateHostedZoneID() != ""` then `PrivateHostedZoneRoleARN()` must have a value.

Reply:
We can alternatively check this condition in the same `if` statement in line `338`.

Reply:
@oriAdler 
I removed the redundant check in the change
Please help to review this PR again, thanks